src/app/utils/Utils.ts

```diff
@@ -925,4 +925,13 @@ export class Utils {
     return typeof value === 'string' ? '"' + value.replace(/"/g, '""') + '"' : value;
   }
 
+  public static removeUnsetAttributes(data: any) {
+    // Remove every undefined, null or empty strings attributes from an object to avoid triggering backend validation with wrong values
+    for (const key of Object.keys(data)) {
```

Comment:
Dangerous as we sometimes want to nullify a property.
It should work without this and null props should be explicitly set.
What do you think?

Reply:
Yep you're right, I'll nullify the values causing problems